### PR TITLE
final design touches plus fixed autocomplete with Mark

### PR DESCRIPTION
**Pull request by mariamgrig**
fixed autocomplete problem, fixed about page, got rid of the awkward index container, fixed the profile item card sizes

### CODE REVIEW

File: app/views/items/index.html.erb [hunk 1]
```diff
@@ -67,14 +81,5 @@
         <% end %>
       </tbody>
     </table>
-    <%= form_tag(item_search_path, :method => "get") do %>
```
Comment (amycsong98) [inline app/views/items/index.html.erb:70]: thanks for fixing this!

File: app/views/layouts/_card_view.html.erb [hunk 2]
```diff
@@ -12,6 +12,5 @@
     <%else%> 
         Not deliverable
     <%end%>
-    <%= item.category %>
```
Comment (amycsong98) [inline app/views/layouts/_card_view.html.erb:15]: why was this removed?

File: app/assets/stylesheets/custom.css.scss [hunk 3]
```diff
@@ -420,4 +415,10 @@ form label {
 	top: 0;
 	z-index: 1;
 	left: 0;
+}
+
+
+.about_class {
```
Comment (amycsong98) [inline app/assets/stylesheets/custom.css.scss:421]: should this be padding? or does margin also work?

**Review by amycsong98 — APPROVED** [review]
LGTM thanks :)

File: app/assets/stylesheets/custom.css.scss (hunk 3 above)
Comment (mrubakh) [reply app/assets/stylesheets/custom.css.scss:421]: margin is fine here

File: app/views/layouts/_card_view.html.erb (hunk 2 above)
Comment (mariamgrig) [reply app/views/layouts/_card_view.html.erb:15]: honestly I removed it coz it was taking a lot of space, then I fixed the size of the container instead but didn't put the category back. also I felt like doesn't really matter much

File: app/assets/stylesheets/custom.css.scss (hunk 3 above)
Comment (mariamgrig) [reply app/assets/stylesheets/custom.css.scss:421]: I think both work!

File: app/views/layouts/_card_view.html.erb (hunk 2 above)
Comment (mrubakh) [reply app/views/layouts/_card_view.html.erb:15]: if it is a clean-look vs. category showing tradeoff, then I would leave it as is

otherwise, category might be nice

**Comment by mrubakh** [comment]
LGTM, let's merge our final PR woooooooo
